reference/docs-conceptual/install/PowerShell-Support-Lifecycle.md

```diff
@@ -112,18 +112,20 @@ use the traditional, paid support options.
 
 ## PowerShell End-of-support dates
 
-Based on these lifecycle policies, the following table lists the dates when currently supported version will no
-longer be supported:
+Based on these lifecycle policies, the following table lists the dates when currently supported
+version will no longer be supported:
 
 |      Version      |   End-of-support   |
 | ----------------- | ------------------ |
+| 7.3 (Stable)      | - |
```

Comment:
@SteveL-MSFT mentioned that support for .NET is 18 months so we may need to adjust these dates.

Reply:
That puts 7.3 at May 8, 2024, do we plan to retroactively modify the dates for 7.1 and earlier?